src/SwiftTaskProvider.ts

```diff
@@ -45,8 +45,8 @@ function createCleanTask(): vscode.Task {
 /**
  * Creates a {@link vscode.Task Task} to run an executable target.
  */
- function createExecutableTask(target: Target): vscode.Task {
-    return createSwiftTask('swift', ['run', target.name], `Run ${target.name}`, vscode.TaskGroup.Build);
+ function createExecutableTask(product: Product): vscode.Task {
```

Comment:
The function name and documentation are out-of-sync with its implementation. Maybe it should be `createBuildTask` now?

Reply:
ok